client/app/visualizations/counter/getCounterData.js

```diff
@@ -0,0 +1,72 @@
+import { isNil, isNumber, isFinite, toString, invoke, extend } from "lodash";
+import numeral from "numeral";
+import { formatSimpleTemplate } from "@/lib/value-format";
+import counterTypes from "./counterTypes";
+
+function formatValue(value, { numberFormat, stringDecChar, stringThouSep }) {
+  if (!isNumber(value)) {
+    return toString(value);
+  }
+
+  // Temporarily update locale data (restore defaults after formatting)
+  const locale = numeral.localeData();
+  const savedDelimiters = locale.delimiters;
+
+  if (stringDecChar || stringThouSep) {
+    locale.delimiters = {
+      thousands: stringThouSep,
+      decimal: stringDecChar || ".",
+    };
+  }
+  const result = numeral(value).format(numberFormat);
+
+  locale.delimiters = savedDelimiters;
+  return result;
+}
+
+function getCounterValue(rows, valueOptions, counterOptions) {
+  const [value, additionalFields] = invoke(counterTypes[valueOptions.type], "getValue", rows, valueOptions);
+
+  if (!valueOptions.show || isNil(value)) {
+    return { value: null, display: null, tooltip: null };
+  }
+
+  const formatData = extend({}, additionalFields, {
+    "@@value": toString(value),
+    "@@value_formatted": isFinite(value) ? formatValue(value, counterOptions) : toString(value),
```

Comment:
@kravets-levko, I couldn't get the `{{ column_name }}` to work neither for counters, nor for Charts. I wonder if it's missing an entry here?

Reply:
Nevermind, I was putting the wrong column name :sweat_smile:.